lib/api/v3/errors/error_representer.rb

```diff
@@ -57,30 +57,7 @@ def _type
         def error_identifier
           return 'urn:openproject-org:api:v3:errors:MultipleErrors' unless Array(represented.errors).empty?
 
-          case represented
-          when ::API::Errors::Conflict
-            'urn:openproject-org:api:v3:errors:UpdateConflict'
-          when ::API::Errors::NotFound
-            'urn:openproject-org:api:v3:errors:NotFound'
-          when ::API::Errors::Unauthenticated
-            'urn:openproject-org:api:v3:errors:Unauthenticated'
-          when ::API::Errors::Unauthorized
-            'urn:openproject-org:api:v3:errors:MissingPermission'
-          when ::API::Errors::UnwritableProperty
-            'urn:openproject-org:api:v3:errors:PropertyIsReadOnly'
-          when ::API::Errors::PropertyFormatError
-            'urn:openproject-org:api:v3:errors:PropertyFormatError'
-          when ::API::Errors::Validation
-            'urn:openproject-org:api:v3:errors:PropertyConstraintViolation'
-          when ::API::Errors::InvalidRenderContext
-            'urn:openproject-org:api:v3:errors:InvalidRenderContext'
-          when ::API::Errors::InvalidUserStatusTransition
-            'urn:openproject-org:api:v3:errors:InvalidUserStatusTransition'
-          when ::API::Errors::InvalidRequestBody
-            'urn:openproject-org:api:v3:errors:InvalidRequestBody'
-          when ::API::Errors::UnsupportedMediaType
-            'urn:openproject-org:api:v3:errors:TypeNotSupported'
-          end
+          represented.class.identifier rescue nil
```

Comment:
The change looks good codewise. However there is one drawback. It makes the formerly not version specific api errors version specific because of the namespace e.g. `urn:openproject-org:api:v3:errors:UpdateConflict`. 

We should either decide that the errors are api version specific and then move their code into the API::V3 errors namespace. Or we decide that they are not and find a different way to refactor this. My beloved way of hiding case statements by introducing a hash comes to mind.